politeiawww/api/www/v1/v1.go

```diff
@@ -61,11 +61,11 @@ const (
 	RouteAllVoteStatus            = "/proposals/votestatus"
 	RouteProposalPaywallDetails   = "/proposals/paywall"
 	RouteProposalPaywallPayment   = "/proposals/paywallpayment"
-	RouteProposalDetails          = "/proposals/{token:[A-z0-9]{64}}"
-	RouteSetProposalStatus        = "/proposals/{token:[A-z0-9]{64}}/status"
-	RouteCommentsGet              = "/proposals/{token:[A-z0-9]{64}}/comments"
-	RouteVoteResults              = "/proposals/{token:[A-z0-9]{64}}/votes"
-	RouteVoteStatus               = "/proposals/{token:[A-z0-9]{64}}/votestatus"
+	RouteProposalDetails          = "/proposals/{token:[A-z0-9]+}"
```

Comment:
Can we make these `[A-z0-9]{7,64}` so that we have a max an min length set.  The new tokens are going to be somewhere between that length.

Reply:
Done